autogpt_platform/backend/backend/api/features/v1_credits_authz_test.py

```
@@ -0,0 +1,302 @@
+org they read/mutate the shared ``OrgBalance``. They must therefore require
+org-level ``MANAGE_BILLING`` (owner or billing_manager) — a plain org member
+must be rejected with 403. Personal-org owners always carry ``is_org_owner``,
+so the gate is a no-op for them.
+
+The gate is applied as an independent per-route dependency (there is no
+shared router-level enforcement), so every gated route is asserted here:
+dropping the dependency from any single route must fail this suite.
+"""
+
+import dataclasses
+from typing import Any, Callable
```

Ruff (0.16.1)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)